src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import { TodoFilter } from '../TodoFilter';
+
+interface Props {
+  filterBy: string;
+  setFilterBy: (filter: string) => void;
+  activeCount: number;
+}
+
+export const Footer: React.FC<Props> = ({
+  filterBy,
+  setFilterBy,
+  activeCount,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeCount} items left`}
+      </span>
+
+      <TodoFilter
+        filterBy={filterBy}
+        setFilterBy={setFilterBy}
+      />
+
+      <button
+        type="button"
+        className={classNames('todoapp__clear-completed', {
+          hidden: activeCount,
+        })}
+        onClick={() => setFilterBy('Active')}
```

Comment:
The onClick event for the 'Clear completed' button should call a function to clear completed todos instead of setting the filter to 'Active'.